kubernetes/configs/dev/one_trust_notices.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+LANGS=it
+PUBLIC_STORAGE_PATH=one-trust-notices
+CONTENT_STORAGE_BUCKET=interop-privacy-notices-content-dev
+HISTORY_STORAGE_BUCKET=interop-privacy-notices-history-dev
```

Comment:
I just noticed: these variables are in a global scope, they should have a comprehensive naming outside of this scope, like
```suggestion
PRIVACY_NOTICES_LANGS=it
PRIVACY_NOTICES_PUBLIC_STORAGE_PATH=one-trust-notices
PRIVACY_NOTICES_CONTENT_STORAGE_BUCKET=interop-privacy-notices-content-dev
PRIVACY_NOTICES_HISTORY_STORAGE_BUCKET=interop-privacy-notices-history-dev
```

Plus, what is `PUBLIC_STORAGE_PATH`?

Reply:
These are global as the value doesn't change per environment, I thought that this was the correct way to handle this cases, isn't it?

@Carminepo2 can you answer for `PUBLIC_STORAGE_PATH`?

Reply:
Not quite, they are global across the deployment of all the environment.
Values that are constant across all environments are in the `shared.sh` file

Reply:
My bad. `PUBLIC_STORAGE_PATH` is not used by the job anymore, it can be removed.